angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.ts

```diff
@@ -220,10 +220,7 @@ export class SprkAccordionItemComponent implements OnInit {
    * @ignore
    */
   getClasses(): string {
-    const classArray: string[] = [
-      'sprk-c-Accordion__item',
-      'sprk-u-Overflow--hidden',
```

Comment:
I didnt see this overflow class doing anything and it wasnt in other frameworks but i want us to pay close attention during testing.

Reply:
confirmed,  I don't think this is doing anything either

Reply:
thank you for double checking @RVMendoza !!!